src/test/java/ch/njol/skript/config/ConfigTest.java

```diff
@@ -24,27 +24,30 @@ public void testUpdateNodes() {
 
 		for (Node node : newNodes) {
 			assertTrue("Node " + node + " was not updated", updatedNodes.contains(node));
-
-			if (node instanceof EntryNode entry) {
-				assertEquals(entry.getValue(), old.get(entry.getPathSteps()));
-			}
 		}
 
 		// keeps removed/user-added nodes
-		assertEquals("true", old.getValue("outdated value"));
+		assertEquals("true", old.get(new String[] {"outdated value"}));
```

Comment:
As I mentioned, we might want to see about renaming one (or both) these `get` methods, as this is rather confusing

Reply:
But I left the old `get` (value getter) for backwards compatibility, in case addons or other things were using it.